RELEASE.md

```diff
@@ -1,7 +1,7 @@
 ## Releasing a new version of Leaflet
 
 - [ ] Ensure all https://github.com/Leaflet/Leaflet/labels/blocker issues and pull requests are resolved.
-- [ ] Update [the changelog](https://github.com/Leaflet/Leaflet/blob/main/CHANGELOG.md) since last release and commit.
+- [ ] Update [the changelog](https://github.com/Leaflet/Leaflet/blob/main/CHANGELOG.md) since last release and commit. Make sure all tests pass on CI before proceeding.
```

Comment:
```suggestion
- [ ] Update [the changelog](https://github.com/Leaflet/Leaflet/blob/main/CHANGELOG.md) since last release and commit.
- [ ] Make sure all tests pass on CI before proceeding.
```